chrysom/basicClient_test.go

```diff
@@ -188,8 +188,9 @@ func TestSendRequest(t *testing.T) {
 			defer server.Close()
 
 			client, err := NewBasicClient(BasicClientConfig{
-				Address: "example.com",
-				Bucket:  "bucket-name",
+				HTTPClient: http.DefaultClient,
```

Comment:
it might be worth making the BasicClientConfig assume a nil HTTPClient means use `http.DefaultClient`